content/docs/ref/ci.md

```diff
@@ -5,12 +5,15 @@ cml ci [options]
 ```
 
 Prepares Git repository for CML operations (setting Git `user.name` &
-`user.email`, unshallow clone and undo CI oddities such as origin URL formatting
-and HTTP remote proxies).
+`user.email`, fetch all branch tips, undo CI oddities such as origin URL
+formatting and HTTP remote proxies, and optionally unshallow clone).
```

Comment:
> _(setting Git `user.name` &_ [...]
```suggestion
`user.email`; fetching all branch tips; undoing CI oddities such as origin URL
formatting and HTTP remote proxies; and, optionally, unshallowing clone).
```

Reply:
Use `\wing` everywhere for consistency with the first line? Also changed some punctuation to separate better the actions.